Worksheet_B/Readme.MD

```diff
@@ -0,0 +1 @@
+![Screenshot](FalloutHackFlow.pdf)
```

Comment:
Flowchart looks good -- nice use of swimlanes. All that is missing is what happens when the player runs out of lives.